packages/ui/src/ERC1155Controls.svelte

```diff
@@ -64,10 +69,17 @@
         Useful for emergency response.
       </HelpTooltip>
     </label>
+    <label class:checked={opts.updatableUri}>
+      <input type="checkbox" bind:checked={opts.updatableUri}>
+      Updatable URI
+      <HelpTooltip link="https://docs.openzeppelin.com/contracts/4.x/api/token/erc1155#ERC1155-_setURI-string-">
+        Privileged accounts will be able to set a new URI for all token types. Clients will replace any instance of &lbrace;id&rbrace; in the URI with the tokenId.
```

Comment:
May be nicer to read? In case you didn't consider it.

```suggestion
        Privileged accounts will be able to set a new URI for all token types. Clients will replace any instance of {"{id}"} in the URI with the tokenId.
```